lib/SwiftDemangle/CMakeLists.txt

```diff
@@ -9,14 +9,12 @@ swift_install_in_component(compiler
     ARCHIVE DESTINATION "lib${LLVM_LIBDIR_SUFFIX}")
 
 # Create a compatibility symlink.
-
-add_custom_command(TARGET swiftDemangle POST_BUILD
-    COMMAND
-      "${CMAKE_COMMAND}" "-E" "create_symlink"
-      "libswiftDemangle.dylib"
-      "libfunctionNameDemangle.dylib"
-    WORKING_DIRECTORY "${SWIFT_LIBRARY_OUTPUT_INTDIR}"
-    COMMENT "Creating compatibility symlink for functionNameDemangle.dylib")
+swift_create_post_build_symlink(swiftDemangle
+  FALSE
+  "libswiftDemangle.dylib"
+  "libfunctionNameDemangle.dylib"
```

Comment:
Is this even needed for Windows? This is really a hack to workaround a library being renamed. I'm not even sure we actually need to create this symlink at all anymore.